packaging/windows/BUILD.txt

```diff
@@ -0,0 +1,120 @@
+How to make a darktable windows installer (64 bit only):
```

Comment:
Did you see that there is already `doc/Windows.txt`? Maybe try merging both into one file – I don't care which place to use.